src/Config/config.json

```diff
@@ -1 +1 @@
-{"test_in_progress": "False", "language": "python", "template": "D:\\Alok country\\Apun coder banega\\Projects\\CP-CLI\\test\\code.cpp", "proxy": ""}
\ No newline at end of file
+{"user": {"firstname": "", "lastname": "", "rating": [""], "contri": [""], "rank": [""], "maxrating": ""}, "test_in_progress": "False", "language": "python", "template": "D:\\Alok country\\Apun coder banega\\Projects\\CP-CLI\\test\\code.cpp", "proxy": ""}
```

Comment:
Why storing rating, contri, rank and maxrating in an array?

Reply:
it will get easier if we call get-user method anywhere else in the code

Reply:
I don't get it. How come it will become easier?

Reply:
I just tried to encapsulate user in one key value pair.
I thought it be easier to call property of user from this key value pair, instead of calling individual properties of user

Reply:
It's a good approach. Probably you are not getting my question. What I an asking is why is your rating property in user object an array? As you can see it's written that "rating": [""]. Why this instead of "rating" : "".

Reply:
I had the same doubt actually, when I call it, it comes in this form only.